app/konnect/architecture.md

```diff
@@ -11,7 +11,7 @@ anywhere, on-premise or in the cloud.
 
 > Figure 1: Diagram of {{site.konnect_short_name}} modules.
 
-Runtime instances, acting as data planes, listen for traffic on the proxy port 443
+Data Plane Nodes, acting as data planes, listen for traffic on the proxy port 443
```

Comment:
```suggestion
Data plane nodes listen for traffic on the proxy port 443
```
I think we can make the executive decision here to use lowercase "data plane nodes", in the same way that we did for "runtime instances".